src/assets/sage.html

```diff
@@ -70,6 +70,28 @@
       var commit    = buildInfo[2];
       var versionString = buildDate + " | " + commit;
 
+      var providers = [
+        "localStorage",
+        {
+          "name": "readOnly",
+          "displayName": "Sage Examples",
+          "src": "examples.json"
+        },
+        {
+          "name": "googleDrive",
+          "apiKey": "AIzaSyAUobrEXqtbZHBvr24tamdE6JxmPYTRPEA",
+          "clientId": "1095918012594-svs72eqfalasuc4t1p1ps1m8r9b8psso.apps.googleusercontent.com"
+        },
+        {
+          "name": "documentStore",
+          "patch": true
+        }
+      ]
+
+      // this is to allow for developer debugging of diagrams
+      if (window.location.search.indexOf("enableLocalFiles") !== -1) {
```

Comment:
Point of potential interest (POPI): I prefer `>= 0` for this test as I think it reads more as "is valid" rather than the double-negative "is not the magic sentinel value that indicates failure".